server/paymentProviders/stripe/creditcard.js

```diff
@@ -216,9 +221,9 @@ const createChargeAndTransactions = async (hostStripeAccount, { order, hostStrip
       amount: order.totalAmount,
       currency: order.currency,
       hostCurrency: balanceTransaction.currency?.toUpperCase(),
-      amountInHostCurrency: balanceTransaction.amount,
-      hostCurrencyFxRate: balanceTransaction.amount / order.totalAmount,
-      paymentProcessorFeeInHostCurrency: fees.stripeFee,
+      amountInHostCurrency: amountInHostCurrency,
```

Comment:
```suggestion
      amountInHostCurrency,
```

Doesn't have to be addressed.

Reply:
Done. 👍🏽